Merge develop into develop-osu

dandahle opened the pull request:
We've made some changes to develop and they all seem to be working now so I'm going to make that the baseline for the master. Also, the develop-osu branch does not allow me to create new users. 

My hope in doing this merge is that it will both solve the user creation issue and show the Orion work in the now-current develop code so we can better test it and get it into production by the AppVetting team.

rarafon commented:
I hope I did the conflict resolution correctly. It was only a conflict with spacing from what I saw.

dandahle commented:
Hi Andy - thank you for doing this. I’m home now and will take a look at it this morning. ==> Dan

> On Jun 18, 2020, at 9:51 PM, Andy Situ <notifications@github.com <mailto:notifications@github.com>> wrote:
> 
> 
> I hope I did the conflict resolution correctly. It was only a conflict with spacing from what I saw.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/dandahle/Catalyst-AppVetting/pull/172#issuecomment-646433164>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ABQDU4O5SKS427QJOYHOLT3RXLVGXANCNFSM4N6MH6JQ>.
>

dandahle commented:
Hi Andy - thank you for doing this. I’m home now and will take a look at it this morning. ==> Dan

> On Jun 18, 2020, at 9:51 PM, Andy Situ <notifications@github.com> wrote:
> 
> 
> I hope I did the conflict resolution correctly. It was only a conflict with spacing from what I saw.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/dandahle/Catalyst-AppVetting/pull/172#issuecomment-646433164>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ABQDU4O5SKS427QJOYHOLT3RXLVGXANCNFSM4N6MH6JQ>.
>

dandahle commented:
Hi Andy - my days didn't go as planned Friday and yesterday, but this morning I got the merge pull and updated the OSU instance at 52.24.42.230. I had some error messages when I did the git pull process. I'm not sure if you have new dependencies in your code from what we had before. Also, a new package-lock.json file was not created, which hasn't happened to me before. 

The good news is that the application runs and I'm now able to create users, but when I tried creating a service in Win10 with Firefox, it required a specific date format with no help about the format required. (Also, the search process is not simplified, but not sure if that's just because those changes have not been made.) 

So in short, I'm not sure if the errors in updating the new code are causing issues with running the application code. 

Could you look through my session log (atached) and see if there's something different I need to do to install the new OSU code? I'll be available later today or tomorrow if you have questions or want to chat about what I did. 

Thank you, 

==> Dan
On 6/20/2020 9:08:56 AM, Justin Bethel <notifications@github.com> wrote:
@bethelju approved this pull request.
This looks good to me. Hopefully the differences in user packages or creating admins may fix the issue
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub [https://github.com/dandahle/Catalyst-AppVetting/pull/172#pullrequestreview-434437971], or unsubscribe [https://github.com/notifications/unsubscribe-auth/ABQDU4MTAANLQZ7W57QM35LRXTNJRANCNFSM4N6MH6JQ].
login as: ubuntu
Authenticating with public key "imported-openssh-key"
Passphrase for key "imported-openssh-key":
Welcome to Ubuntu 18.04.4 LTS (GNU/Linux 5.3.0-1019-aws x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage

  System information as of Sun Jun 21 15:19:27 UTC 2020

  System load:  0.0                Processes:           99
  Usage of /:   15.1% of 29.02GB   Users logged in:     0
  Memory usage: 39%                IP address for eth0: 172.31.27.40
  Swap usage:   0%

 * "If you've been waiting for the perfect Kubernetes dev solution for
   macOS, the wait is over. Learn how to install Microk8s on macOS."

   https://www.techrepublic.com/article/how-to-install-microk8s-on-macos/

 * Canonical Livepatch is available for installation.
   - Reduce system reboots and improve kernel security. Activate at:
     https://ubuntu.com/livepatch

17 packages can be updated.
0 updates are security updates.

Failed to connect to https://changelogs.ubuntu.com/meta-release-lts. Check your Internet connection or proxy settings


*** System restart required ***
Last login: Mon Jun  8 05:38:29 2020 from 162.229.194.81
ubuntu@ip-172-31-27-40:~$ cd   /usr/src/Catalyst-AppVetting
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ ls
CONTRIBUTING.md   bin                           docker-compose.yml  restart.sh
Dockerfile        carenetwork                   models              routes
LICENSE.md        controllers                   mongoose            script
MongoUpdate.md    db_backup.sh                  node_modules        test
README-DOCKER.md  db_backup_delete.sh           package-lock.json   views
README.md         db_backup_develop.sh          package.json
SECURITY.md       db_backup_restore.sh          passport
app.js            db_backup_restore_develop.sh  public
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ sudo ./script/stop-all.sh
info:    Forever stopped processes:
data:        uid  command       script  forever pid   id logfile                        uptime
data:    [0] hSoC /usr/bin/node bin/www 10098   10110    /home/ubuntu/.forever/hSoC.log 13:9:35:34.445999999996275
./script/stop-all.sh: line 6:  7516 Terminated              ./script/stop-mongod.sh
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ sudo git branch
* develop-osu
  develop-osu-reporting
  master
  origin/develop-osu
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ sudo npm run pull

> catalyst-appvetting-tool@1.0.0 pull /usr/src/Catalyst-AppVetting
> git pull --no-edit

remote: Enumerating objects: 100, done.
remote: Counting objects: 100% (100/100), done.
remote: Compressing objects: 100% (51/51), done.
remote: Total 100 (delta 54), reused 79 (delta 47), pack-reused 0
Receiving objects: 100% (100/100), 104.83 KiB | 4.03 MiB/s, done.
Resolving deltas: 100% (54/54), completed with 14 local objects.
From https://github.com/dandahle/Catalyst-AppVetting
   801bc09..ccb1bfd  develop-osu     -> origin/develop-osu
   b4efaec..3d44563  care-network    -> origin/care-network
   e915700..fe1bdc2  develop         -> origin/develop
 * [new branch]      safari-date-fix -> origin/safari-date-fix
Updating 801bc09..ccb1bfd
error: Your local changes to the following files would be overwritten by merge:
        package-lock.json
Please commit your changes or stash them before you merge.
Aborting
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! catalyst-appvetting-tool@1.0.0 pull: `git pull --no-edit`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the catalyst-appvetting-tool@1.0.0 pull script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-06-21T15_21_44_151Z-debug.log
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ rm package-lock.json
rm: remove write-protected regular file 'package-lock.json'? y
rm: cannot remove 'package-lock.json': Permission denied
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ sudo rm package-lock.json
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ sudo git pull
Updating 801bc09..ccb1bfd
Fast-forward
 controllers/api.js               | 68 ++++++++++++++++----------------
 models/documentPackage.js        |  3 +-
 models/userPackage.js            |  2 +-
 package-lock.json                |  3 +-
 public/javascripts/orderTable.js | 43 ++++++++++++++++++--
 routes/partners.js               |  8 +---
 routes/view.js                   | 23 +++++++----
 script/createAdminUser.js        |  2 +-
 views/b3-view.hbs                | 78 ++++++++++++++++++++++++++++++++----
 views/b3-worksheet-view.hbs      | 85 +++++++++++++++++++++++++++++++++++++---
 views/exportPDF.hbs              |  9 ++++-
 views/projectview.hbs            |  2 +-
 views/vetting.hbs                | 10 ++---
 13 files changed, 256 insertions(+), 80 deletions(-)
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ sudo npm update
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.1 (node_modules/mocha/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN The package minimist is included as both a dev and production dependency.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.12 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.12: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

+ express-session@1.17.1
+ jquery@3.5.1
+ mocha@7.2.0
+ nyc@15.1.0
+ mongodb@3.5.9
+ mongoose@5.9.19
added 1 package from 1 contributor, removed 38 packages, updated 26 packages and audited 790 packages in 11.249s

30 packages are looking for funding
  run `npm fund` for details

found 4 low severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details


   ╭────────────────────────────────────────────────────────────────╮
   │                                                                │
   │      New patch version of npm available! 6.14.4 → 6.14.5       │
   │   Changelog: https://github.com/npm/cli/releases/tag/v6.14.5   │
   │               Run npm install -g npm to update!                │
   │                                                                │
   ╰────────────────────────────────────────────────────────────────╯

ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ sudo npm install minimist --save-dev
npm notice save minimist is being moved from dependencies to devDependencies
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.12 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.12: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/mocha/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

+ minimist@1.2.5
added 1 package from 1 contributor, updated 1 package and audited 790 packages in 6.217s

37 packages are looking for funding
  run `npm fund` for details

found 4 low severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ sudo rm package-lock.json
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ sudo npm install
npm WARN lifecycle catalyst-appvetting-tool@1.0.0~preinstall: cannot run in wd catalyst-appvetting-tool@1.0.0 npx npm-force-resolutions (wd=/usr/src/Catalyst-AppVetting)
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

> puppeteer@3.3.0 install /usr/src/Catalyst-AppVetting/node_modules/puppeteer
> node install.js

ERROR: Failed to set up Chromium r756035! Set "PUPPETEER_SKIP_DOWNLOAD" env variable to skip download.
Error: EACCES: permission denied, mkdir '/usr/src/Catalyst-AppVetting/node_modules/puppeteer/.local-chromium'
  -- ASYNC --
    at BrowserFetcher.<anonymous> (/usr/src/Catalyst-AppVetting/node_modules/puppeteer/lib/helper.js:94:19)
    at fetchBinary (/usr/src/Catalyst-AppVetting/node_modules/puppeteer/install.js:148:8)
    at download (/usr/src/Catalyst-AppVetting/node_modules/puppeteer/install.js:54:9) {
  errno: -13,
  code: 'EACCES',
  syscall: 'mkdir',
  path: '/usr/src/Catalyst-AppVetting/node_modules/puppeteer/.local-chromium'
}
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.2.7 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.1 (node_modules/mocha/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! puppeteer@3.3.0 install: `node install.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the puppeteer@3.3.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-06-21T15_25_47_794Z-debug.log
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ sudo npm install --unsafe-perm

> catalyst-appvetting-tool@1.0.0 preinstall /usr/src/Catalyst-AppVetting
> npx npm-force-resolutions

npx: installed 5 in 1.349s
ENOENT: no such file or directory, open './package-lock.json'
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! catalyst-appvetting-tool@1.0.0 preinstall: `npx npm-force-resolutions`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the catalyst-appvetting-tool@1.0.0 preinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-06-21T15_26_20_321Z-debug.log
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ sudo npm update
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ npm ls minmist
catalyst-appvetting-tool@1.0.0 /usr/src/Catalyst-AppVetting
└── (empty)

ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ ls
CONTRIBUTING.md   controllers                   package.json
Dockerfile        db_backup.sh                  passport
LICENSE.md        db_backup_delete.sh           public
MongoUpdate.md    db_backup_develop.sh          restart.sh
README-DOCKER.md  db_backup_restore.sh          routes
README.md         db_backup_restore_develop.sh  script
SECURITY.md       docker-compose.yml            test
app.js            models                        views
bin               mongoose
carenetwork       node_modules
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ sudo npm install --unsafe-perm

> catalyst-appvetting-tool@1.0.0 preinstall /usr/src/Catalyst-AppVetting
> npx npm-force-resolutions

npx: installed 5 in 1.343s
ENOENT: no such file or directory, open './package-lock.json'
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! catalyst-appvetting-tool@1.0.0 preinstall: `npx npm-force-resolutions`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the catalyst-appvetting-tool@1.0.0 preinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-06-21T15_29_54_348Z-debug.log
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ sudo npx npm-force-resolutions
npx: installed 5 in 1.307s
ENOENT: no such file or directory, open './package-lock.json'
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ ls minimist
ls: cannot access 'minimist': No such file or directory
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ npm ls minimist
catalyst-appvetting-tool@1.0.0 /usr/src/Catalyst-AppVetting
├─┬ forever@2.0.0
│ ├─┬ flatiron@0.4.3
│ │ ├─┬ broadway@0.3.6
│ │ │ └─┬ nconf@0.6.9
│ │ │   └─┬ optimist@0.6.0
│ │ │     └── minimist@0.0.10
│ │ └─┬ optimist@0.6.0
│ │   └── minimist@0.0.10
│ ├─┬ mkdirp@0.5.5
│ │ └── minimist@1.2.5
│ ├─┬ optimist@0.6.1
│ │ └── minimist@0.0.10
│ └─┬ prettyjson@1.2.1
│   └── minimist@1.2.5
├─┬ hbs@4.1.1
│ └─┬ handlebars@4.7.6
│   └── minimist@1.2.5
├── minimist@1.2.5
├─┬ nyc@15.1.0
│ └─┬ istanbul-lib-instrument@4.0.3
│   └─┬ @babel/core@7.10.3
│     └─┬ json5@2.1.3
│       └── minimist@1.2.5  deduped
└─┬ pdfmake@0.1.65
  └─┬ linebreak@1.0.2
    └─┬ brfs@2.0.2
      └─┬ quote-stream@1.0.2
        └── minimist@1.2.5


┌──────────────────────────────────────────────────────────┐
│                 npm update check failed                  │
│           Try running with sudo or get access            │
│           to the local update config store via           │
│ sudo chown -R $USER:$(id -gn $USER) /home/ubuntu/.config │
└──────────────────────────────────────────────────────────┘
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ sudo npm ls minimist
catalyst-appvetting-tool@1.0.0 /usr/src/Catalyst-AppVetting
├─┬ forever@2.0.0
│ ├─┬ flatiron@0.4.3
│ │ ├─┬ broadway@0.3.6
│ │ │ └─┬ nconf@0.6.9
│ │ │   └─┬ optimist@0.6.0
│ │ │     └── minimist@0.0.10
│ │ └─┬ optimist@0.6.0
│ │   └── minimist@0.0.10
│ ├─┬ mkdirp@0.5.5
│ │ └── minimist@1.2.5
│ ├─┬ optimist@0.6.1
│ │ └── minimist@0.0.10
│ └─┬ prettyjson@1.2.1
│   └── minimist@1.2.5
├─┬ hbs@4.1.1
│ └─┬ handlebars@4.7.6
│   └── minimist@1.2.5
├── minimist@1.2.5
├─┬ nyc@15.1.0
│ └─┬ istanbul-lib-instrument@4.0.3
│   └─┬ @babel/core@7.10.3
│     └─┬ json5@2.1.3
│       └── minimist@1.2.5  deduped
└─┬ pdfmake@0.1.65
  └─┬ linebreak@1.0.2
    └─┬ brfs@2.0.2
      └─┬ quote-stream@1.0.2
        └── minimist@1.2.5

ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ npm audit
npm ERR! code EAUDITNOLOCK
npm ERR! audit Neither npm-shrinkwrap.json nor package-lock.json found: Cannot audit a project without a lockfile
npm ERR! audit Try creating one first with: npm i --package-lock-only

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-06-21T15_33_42_444Z-debug.log
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ sudo git branch
* develop-osu
  develop-osu-reporting
  master
  origin/develop-osu
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ sudo git pull
Already up to date.
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ sudo npm update
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ ls
CONTRIBUTING.md   controllers                   package.json
Dockerfile        db_backup.sh                  passport
LICENSE.md        db_backup_delete.sh           public
MongoUpdate.md    db_backup_develop.sh          restart.sh
README-DOCKER.md  db_backup_restore.sh          routes
README.md         db_backup_restore_develop.sh  script
SECURITY.md       docker-compose.yml            test
app.js            models                        views
bin               mongoose
carenetwork       node_modules
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ sudo reboot
login as: ubuntu
Authenticating with public key "imported-openssh-key"
Passphrase for key "imported-openssh-key":
Welcome to Ubuntu 18.04.4 LTS (GNU/Linux 5.3.0-1023-aws x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage

  System information as of Sun Jun 21 15:36:02 UTC 2020

  System load:  0.25               Processes:           97
  Usage of /:   13.9% of 29.02GB   Users logged in:     0
  Memory usage: 16%                IP address for eth0: 172.31.27.40
  Swap usage:   0%

 * "If you've been waiting for the perfect Kubernetes dev solution for
   macOS, the wait is over. Learn how to install Microk8s on macOS."

   https://www.techrepublic.com/article/how-to-install-microk8s-on-macos/

 * Canonical Livepatch is available for installation.
   - Reduce system reboots and improve kernel security. Activate at:
     https://ubuntu.com/livepatch

17 packages can be updated.
0 updates are security updates.

Failed to connect to https://changelogs.ubuntu.com/meta-release-lts. Check your Internet connection or proxy settings


Last login: Sun Jun 21 15:19:28 2020 from 50.38.53.207
ubuntu@ip-172-31-27-40:~$ cd /usr/src/Catalyst-AppVetting
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ ls
CONTRIBUTING.md   controllers                   package.json
Dockerfile        db_backup.sh                  passport
LICENSE.md        db_backup_delete.sh           public
MongoUpdate.md    db_backup_develop.sh          restart.sh
README-DOCKER.md  db_backup_restore.sh          routes
README.md         db_backup_restore_develop.sh  script
SECURITY.md       docker-compose.yml            test
app.js            models                        views
bin               mongoose
carenetwork       node_modules
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ sudo ./script/restart-all.sh
info:    No forever processes running
./script/stop-all.sh: line 6:  1370 Terminated              ./script/stop-mongod.sh
2020-06-21T15:36:53.121+0000 I CONTROL  [main] log file "/usr/src/logs/mongod.log" exists; moved to "/usr/src/logs/mongod.log.2020-06-21T15-36-53".
warn:    --minUptime not set. Defaulting to: 1000ms
warn:    --spinSleepTime not set. Your script will exit if it does not stay up for at least 1000ms
info:    Forever processing file: ./bin/www
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$ ls
CONTRIBUTING.md   controllers                   package.json
Dockerfile        db_backup.sh                  passport
LICENSE.md        db_backup_delete.sh           public
MongoUpdate.md    db_backup_develop.sh          restart.sh
README-DOCKER.md  db_backup_restore.sh          routes
README.md         db_backup_restore_develop.sh  script
SECURITY.md       docker-compose.yml            test
app.js            models                        views
bin               mongoose
carenetwork       node_modules
ubuntu@ip-172-31-27-40:/usr/src/Catalyst-AppVetting$

rarafon commented:
Hi Dan,
I've moved to another computer and finally got to transferring the key for the instance.

From what I saw in the instance, the git pull was successful according to git, but it changes made in the instance that isn't present in the branch:
 deleted:    .env.template
 deleted:    package-lock.json
 modified:   package.json
modified:   test/frontend/index.js

I haven't installed any new dependencies as I don't like doing that. 

So someone's been modifying or coding the files on the instance directly, which is weird. The person even deleted package-lock.json, which is even weirder.


For the date format, I implemented the change in the care network branch last week I think so the date and time should be working. I will update that instance.

dandahle commented:
Hi Andy - thank you for looking into this. Some of what you saw was me and some, I think, were changes in develop. 

Through the AppVetting code update process we developed the last couple of years, I’ve been deleting the package-lock.json file because it prevents updates unless you stash and restore it, and from what I understood it just gets rebuilt again when npm install is run. However, this time it didn’t, which was one of my concerns. 

I’m a little surprise the .env.template was deleted, unless it’s deleted after the .env file is created. I’ve forgotten now how Rohin set that up. Because the Access Keys are included in that file but only needed when the instance is initially set up, I’ve asked Rohin to eliminate the .env file in set up, or at least limit the info stored in there to info that the AppVetting tool and MongoDB need to run, or is at least not sensitive. 

Although not intentional, this shows the need for some consistent, knowledgable volunteer help to maintain the tool and keep things runnings. I’m a little out of my depth when something unexpected happens. :-) 

I will look at the Care Network today and also work on cleaning up the development environments. 

Thank you, 

==> Dan

PS: how is your job search going? Is there anything I can do to help? 

> On Jun 22, 2020, at 9:08 PM, Andy Situ <notifications@github.com> wrote:
> 
> 
> Hi Dan,
> I've moved to another computer and finally got to transferring the key for the instance.
> 
> From what I saw in the instance, the git pull was successful according to git, but it changes made in the instance that isn't present in the branch:
> deleted: .env.template
> deleted: package-lock.json
> modified: package.json
> modified: test/frontend/index.js
> 
> I haven't installed any new dependencies as I don't like doing that.
> 
> So someone's been modifying or coding the files on the instance directly, which is weird. The person even deleted package-lock.json, which is even weirder.
> 
> For the date format, I implemented the change in the care network branch last week I think so the date and time should be working. I will update that instance.
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub <https://github.com/dandahle/Catalyst-AppVetting/pull/172#issuecomment-647894926>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ABQDU4LBW6MHZYQYXOPBBKTRYATDDANCNFSM4N6MH6JQ>.
>
